src/main/resources/application.yml

```diff
@@ -40,8 +40,12 @@ app:
     oauth-client:
       resource: "${app.indexing-service.base-uri}"
     publisher-catalog-name: publisher_data
-  query-cleanup-interval: "*/10 * * ? * *" # Every 10 seconds
-  query-cleanup-timeout-in-seconds: 120
+  query-cleanup:
+    cron-interval: "*/10 * * ? * *" # Every 10 seconds
+    query-timeout-in-seconds: 120
```

Comment:
Since you have `query-cleanup` as the parent key, how about we use `timeout-in-seconds` as the key here?

Reply:
Done